SQL CODE.txt

```diff
@@ -0,0 +1,128 @@
+
+
+
+
+CREATE TABLE users
+(
+	userId INT PRIMARY KEY IDENTITY(1,1),
+	first_name VARCHAR(40) NOT NULL,
+	last_name VARCHAR(40) NULL,
+	username VARCHAR(25) UNIQUE NOT NULL,
+	email VARCHAR(30) UNIQUE NOT NULL CHECK (email like '%_@%.com'),
+	[password] VARCHAR(15) NOT NULL,

```

Comment:
why did you put in [password]? if you can't do password, can you have another column name like user_password?